Mathlib/Algebra/Order/Ring/Archimedean.lean

```diff
@@ -197,6 +197,26 @@ theorem mk_ofNat {n : ℕ} [n.AtLeastTwo] : mk (ofNat(n) : S) = 0 :=
 theorem mk_natCast_nonneg (n : ℕ) : 0 ≤ mk (n : S) :=
   mod_cast mk_intCast_nonneg n
 
+theorem exists_mem_Icc_of_mk_nonneg {S : Type*} [Ring S]
```

Comment:
Thinking about it, this should probably be rewritten as `exists_nat_(gt/lt/ge/le)_of_mk_nonneg`.